packages/core/src/migration/utils.ts

```diff
@@ -0,0 +1,21 @@
+import semver from 'semver';
+
+export const migrationFileNameRegex = /^(((?!next).)*)\.js$/;
+
+export const getVersionFromFileName = (fileName: string) => {
+  const match = migrationFileNameRegex.exec(fileName);
```

Comment:
I'm wondering can we get version from package.json?

Reply:
No, these versions are "history" versions